docs/source/crs/projections.rst

```diff
@@ -384,3 +384,18 @@ SouthPolarStereo
     ax.gridlines()
 
 
+Sinusoidal
```

Comment:
OK, this is not in the correct place, but it looks like there are some precision changes in the other projections. We should probably update them, but maybe not in this PR?